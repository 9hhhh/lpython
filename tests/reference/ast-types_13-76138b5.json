{
    "basename": "ast-types_13-76138b5",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/types_13.f90",
    "infile_hash": "21e16ce325d0e643477dd5b1fdb7d8ba34d5e5ccb6be0f79cae6abda",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-types_13-76138b5.stdout",
    "stdout_hash": "fdbafe522f9b69389da07d3ac1644aa7863c6ce61eaf37ec8e776490",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}