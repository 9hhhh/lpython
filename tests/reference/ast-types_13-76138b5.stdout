(TranslationUnit [(Program types_13 [(Use iso_c_binding [(UseSymbol c_int ()) (UseSymbol c_double ()) (UseSymbol c_char ())])] [(Interface () [])] [] [])])
